compel/plugins/shmem/shmem.c

```diff
@@ -4,7 +4,6 @@
 #include <compel/plugins/shmem.h>
 #include <compel/plugins/std/syscall.h>
 #include "shmem.h"
-#include "std-priv.h"
```

Comment:
It seems like we use `std_ctl_sock()` below in `shmem_create()`:
```
ret = sys_write(std_ctl_sock(), &spi, sizeof(spi));
```

Reply:
Right you are.
It seems like we don't build the shmem plugin either:
```
compel-plugins		+= compel/plugins/std.lib.a compel/plugins/fds.lib.a
```
I need to think about what we need from all this and probably re-use it for it's dead now too..